conf/routes

```diff
@@ -40,6 +40,18 @@ POST   /v3/participants/:userId/resendEmailVerification      @org.sagebionetwork
 POST   /v3/participants/:userId/consents/withdraw            @org.sagebionetworks.bridge.play.controllers.ParticipantController.withdrawFromAllConsents(userId: String)
 POST   /v3/participants/:userId/consents/:guid/resendConsent @org.sagebionetworks.bridge.play.controllers.ParticipantController.resendConsentAgreement(userId: String, guid: String)
 
+# Shared Module Metadata
+GET    /v3/sharedmodules/metadata                            @org.sagebionetworks.bridge.play.controllers.SharedModuleMetadataController.getAllMetadataAllVersions
+POST   /v3/sharedmodules/metadata                            @org.sagebionetworks.bridge.play.controllers.SharedModuleMetadataController.createMetadata
+GET    /v3/sharedmodules/metadata/byId/:id                   @org.sagebionetworks.bridge.play.controllers.SharedModuleMetadataController.getMetadataByIdAllVersions(id: String)
+DELETE /v3/sharedmodules/metadata/byId/:id                   @org.sagebionetworks.bridge.play.controllers.SharedModuleMetadataController.deleteMetadataByIdAllVersions(id: String)
+GET    /v3/sharedmodules/metadata/byId/:id/latest            @org.sagebionetworks.bridge.play.controllers.SharedModuleMetadataController.getMetadataByIdLatestVersion(id: String)
+GET    /v3/sharedmodules/metadata/byId/:id/versions/:version @org.sagebionetworks.bridge.play.controllers.SharedModuleMetadataController.getMetadataByIdAndVersion(id: String, version: Int)
+POST   /v3/sharedmodules/metadata/byId/:id/versions/:version @org.sagebionetworks.bridge.play.controllers.SharedModuleMetadataController.updateMetadata(id: String, version: Int)
+DELETE /v3/sharedmodules/metadata/byId/:id/versions/:version @org.sagebionetworks.bridge.play.controllers.SharedModuleMetadataController.deleteMetadataByIdAndVersion(id: String, version: Int)
```

Comment:
byId is redundant here, I would remove it for consistency. Me personally, I would have /v3/sharedmodules/metadata return everything, *unless* you provided tags or query parameters, in which case it would return a qualified subset like /v3/sharedmodules/metadata/query does. But having a /query is okay as a kind of verb modifier.